tests/DevApps/WebAppCallsMicrosoftGraph/appsettings.json

```diff
@@ -17,15 +17,15 @@
             //    "SourceType": "SignedAssertionFromManagedIdentity",
```

Comment:
@trwalke - Did you actually get the CIAM web app to get a token and call Graph? I am actually unsure if that scenario works today (it used to not work).

Reply:
Yes, it worked when testing locally
![image](https://github.com/AzureAD/microsoft-identity-web/assets/30090357/9dfc949b-a480-4ff3-8970-cbd218677dc0)

Reply:
however, see https://github.com/AzureAD/microsoft-identity-web/issues/2732